component/main.jsonnet

```
@@ -22,11 +24,11 @@ local anyControllerUsesAcme = std.foldl(
       metadata+: {
         namespace: params.namespace + '-operator',
       },
-      spec: {
+      spec: if hasAcmeSupport then {
         defaultCertificate: {
```

If I remember correctly we set on preferring conditional keys?

```suggestion
        [if hasAcmeSupport then defaultCertificate]: {
```